jobs/v3/snippets/basic_company_sample.go

```diff
@@ -0,0 +1,190 @@
+// Copyright 2018 Google Inc. All rights reserved.
+// Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+
+package main
```

Comment:
Snippets do not need a package main. Users mostly read this code from cloud.google.com then copy & paste as needed, rather than running it directly. This can be `package snippets` and the `func main`s can be deleted.